otter/convergence/service.py

```diff
@@ -293,45 +279,39 @@ def get_my_divergent_groups(my_buckets, all_buckets):
     :returns: list of dicts, where each dict has ``tenant_id``,
         ``group_id``, and ``version`` keys.
```

Comment:
I realize I missed this documentation referring to `version` -- I've fixed it in an upcoming PR to refer to `dirty-flag`.